source/features/ci-link.tsx

```diff
@@ -25,7 +25,7 @@ async function init(): Promise<false | void> {
 	}
 
 	// Append to title (aware of forks and private repos)
-	appendBefore('.pagehead h1', '.fork-flag', icon);
+	appendBefore('.pagehead h1, .hide-full-screen h1', '.fork-flag, .hide-full-screen span.text-small', icon);
```

Comment:
The DOM completely changed, `.fork-flat` is not inside `h1`.

You can probably just use `select([itemprop="name"])!.after(icon)`